metricbeat/track.json

```diff
@@ -0,0 +1,31 @@
+{% import "rally.helpers" as rally with context %}
+{
+  "version": 2,
+  "description": "Metricbeat data",
+  "indices": [
+    {
+      "name": "metricbeat",
+      "body": "index.json",
+      "types": ["_doc"]
```

Comment:
As types have been removed on master you can just remove this property (the most recent version of Rally will allow you to do this).